patch/flash_appbuilder.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/flask_appbuilder/security/manager.py b/flask_appbuilder/security/manager.py
+index e2b361c9..a454d8c6 100644
+--- a/flask_appbuilder/security/manager.py
```

Comment:
Should the file name to be `flask_appbuilder.patch` rather than `flash_...`?

Reply:
Ooops, indeed. I pushed the fix.